<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgZipRegionCityEdit</class>
 <widget class="QDialog" name="dlgZipRegionCityEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblRegion">
       <property name="text">
        <string>Region :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cmbRegion">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblZip">
       <property name="text">
        <string>Zip :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ledZip">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblCity">
       <property name="text">
        <string>City :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="ledCity">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="btbButtons">
     <property name="standardButtons">
      <set>QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ledZip</tabstop>
  <tabstop>ledCity</tabstop>
  <tabstop>btbButtons</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btbButtons</sender>
   <signal>accepted()</signal>
   <receiver>dlgZipRegionCityEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btbButtons</sender>
   <signal>rejected()</signal>
   <receiver>dlgZipRegionCityEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
